### DISCUSSION TITLE
[功能]: 希望支持自制的agent

### DISCUSSION THREAD

**Discussion by peanutsugarlove** [CherryHQ/cherry-studio#2267] [Ideas]
### 您的功能建议是否与某个问题相关?

单个model能力通常非常有限

### 请描述您希望实现的解决方案

希望支持由python代码或者js代码编写的agnet

**Comment by kangfenmao** [comment]
能否展开聊聊

**Comment by ZouHuaYi** [comment]
是啊加一个自定义的 agent 插件，这样就好很多了，更加灵活。
